ui/StatusQ/src/StatusQ/Controls/StatusPinInput.qml

```diff
@@ -169,6 +177,23 @@ Item {
         }
     }
 
+    /*
+        \qmlmethod StatusPinInput::clearPin()
+
+        Sets the pin input to an empty string, setting state of each digit to "EMPTY", and stops the blinking animation
+
+        Doesn't change the current `pinLen`.
+    */
+    function clearPin() {
+        inputText.text = ""
+        d.currentPinIndex = 0
+        d.deactivateBlink()
+        for (var i = 0; i < root.pinLen; i++) {
+            const currItem = repeater.itemAt(i)
+            currItem.innerState = "EMPTY"
```

Comment:
`EMPTY` should probably be put in a Constant since I assume it's reused/hardcoded somewhere else too